pytorch_lightning/loggers/wandb.py

```diff
@@ -132,11 +132,8 @@ def log_metrics(self, metrics: Dict[str, float], step: Optional[int] = None) ->
 
     @property
     def name(self) -> str:
-        # don't create an experiment if we don't have one
-        name = self._experiment.project_name() if self._experiment else None
-        return name
+        return self.experiment.project_name()
```

Comment:
won't this break in DDP?
if we remove this, every process will try to create the experiment multiple times.

Reply:
@jeremyjordan said he did that to fix a test. Let me try to revert it, maybe it was already fixed by other changes in the master.

Reply:
tests pass, so let's keep it as it is on master. 👍